### ISSUE TITLE
Build and style BGS groups landing page

### ISSUE THREAD

**Issue by angievanhegan** [MantaRayMedia/bgs#133]
# Done criteria for BGS and Manta Ray
 # Build
- [x] **ISSUE 1** @StephMSlack to confirm if links to external sites are required - Yes
- [x] Build page with view of groups card content (grouped by type of group), sorted alphabetically. 'Professions' groups at top and SIGs below (opposite to the wirefames)
- [x] Regional / national groups not shown on this page (but under about)
- [ ] Cards need to link to external site if an external link is specified otherwise to the group page This may not be required Steph to confirm) - Yes link to an external site
- [x] Section label as shown for certain SIGs in the wireframe is no longer required

# Style
- [ ] Inherit existing syles, use card content and exisiting header styles for sections

# Designs and Drupal links
- [Wireframes on RT board](https://realtimeboard.com/app/board/o9J_k0NkHnc=/?moveToWidget=3074457345977318149)
- [Design for the top of the page only](https://invis.io/6RFMRDCV2GN)
- [Drupal page](http://bgs-dev.mrmdev.co.uk/bgs-groups)

**Comment by ican-icant** [comment]
Blocked - waiting on member civi and design of feeds at bottom of page are not yet built

**Comment by angievanhegan** [comment]
@ican-icant there's no feed on this page. Can you explain?

**Comment by angievanhegan** [comment]
@StephMSlack can you confirm if you need to be able to have some box links go to external sites?

**Comment by angievanhegan** [comment]
@dilwarhussain We do need box links to link to external sites (a few of the groups have their own site). There's a done criteria for this at the top. Thanks

**Comment by dilwarhussain** [comment]
Hi @StephMSlack Could you check this is working for you. I've added a new field to use if you'd like to link off to an external site. If not used, default behaviour is to go to the node.

**Comment by StephMSlack** [comment]
@dilwarhussain happy to check but I don't seem to have editing access to the groups function to see what is happening in Drupal.

**Comment by dilwarhussain** [comment]
@StephMSlack Sorry about that, I've changed permissions if you could try again please. Thanks.

**Comment by angievanhegan** [comment]
@dilwarhussain I've put this one back into the sprint backlog with a new issue at the top (2) to make sure we have all the parts on the page that are on the wireframe. Thanks

**Comment by angievanhegan** [comment]
@StephMSlack Can you confirm whether or not you want the section label on here too. Thanks

**Comment by StephMSlack** [comment]
@angie just looking at the groups page - we will need to be able to add content/images potentially beyond the feeds. Can the additional or top content have the standard content nuggets enabled as well (and possibly the box links?)
The type of content could be:
- Mini article specific to the SIG - eg the Diabetes SIG has a session at the BGS conference in April, here is what we are discussing and why, we need your help on X, dont forget to book your ticket to the conference. etc. 
- Call for members to feed into a consultation, where the consultation request requires some explanatory text and information to download
- report on an event that has been attended by SIG member (which would be too specific for a general news article)

**Comment by StephMSlack** [comment]
@angievanhegan - no we do not want the section label on these boxlinks

**Comment by StephMSlack** [comment]
@angievanhegan Sorry - looking at the landing page, there no explanation about the groups - who they are and why they exist. I think we need to be able to add a few lines of content (as per blog and Policy landing pages) to provide more information.

**Comment by StephMSlack** [comment]
@angievanhegan also the cards are not rendering very well in safari - 
<img width="1440" alt="screen shot 2018-03-13 at 16 44 14" src="https://user-images.githubusercontent.com/32127102/37356402-c948ff0a-26dd-11e8-82cc-62008e1a7c2f.png">
